src/Components/Nav.js

```diff
@@ -1,8 +1,48 @@
-import React, { Component } from 'react';
+import React, { Component, useEffect, useState } from 'react';
+import { withRouter, Link } from 'react-router-dom';
 import styled from 'styled-components';
 import Button, { RgbaBtn } from './Button';
 
-const Nav = () => {
+const Nav = ({ history }) => {
+  const [isLogin, setIsLogin] = useState(true);
+  const [profileImg, setProfileImg] = useState('');
+
+  const logout = () => {
+    fetch('http://3.34.199.216:8000/users/logout', {
+      method: 'POST',
+      body: JSON.stringify({
+        access_token: sessionStorage.getItem('Watta_token'),
+      }),
+    })
+      .then(res => res.json())
+      .then(sessionStorage.removeItem('Watta_token'), setIsLogin(!isLogin));
+  };
+
+  const goToLogin = () => {
+    history.push('/login');
+  };
+
+  const goToSignUp = () => {
+    history.push('/login/signup');
+  };
+
+  useEffect(() => {
+    fetch('http://3.34.199.216:8000/users/userinfo', {
+      method: 'GET',
+      headers: {
+        Authorization: sessionStorage.getItem('Watta_token'),
+      },
+    })
+      .then(res => res.json())
+      .then(res =>
+        res.message === 'login_required'
+          ? alert('로그인을 해주세요')
+          : setProfileImg(res.user_info.profile_url)
+      );
```

Comment:
```suggestion
      .then(res => {
        if (res.message === 'login_required') {
          alert('로그인을 해주세요')
          return;
        }

        setProfileImg(res.user_info.profile_url)
      });
```
삼항연산자로 처리하면 한 뎁스의 참/거짓 밖에 표현하지 못할 것 같습니다. (경우의 수가 추가 되면 복잡해져서)
if문으로 처리해주세요!